internals/reminders_test.py

```diff
@@ -58,13 +79,31 @@ def setUp(self):
     self.current_milestone_info = {
         'earliest_beta': '2022-09-21T12:34:56',
     }
-    self.feature_template = core_models.Feature(
-      name='feature one', summary='sum', owner=['feature_owner@example.com'],
-      category=1, ot_milestone_desktop_start=100)
-    self.feature_template.key = ndb.Key('Feature', 123)
-    # This test does not require saving to the database.
+
+    self.old_feature_template = Feature(id=123,
```

Comment:
Where is `old_feature_template` used?

Reply:
Ahh. I understand now.
I think this would be better by being `self.feature_template` and the other one being `self.feature_entry_template`. When I saw `old`, I thought it was a mistake since I saw they both had 123 as the id.  
However, this [thread](https://github.com/GoogleChrome/chromium-dashboard/pull/2517/files#r1035108814) and your TODO gave me context around FeatureEntry becoming the new way.